developer/provider/accessdbprovidersample01.md

```diff
@@ -10,29 +10,20 @@ ms.assetid: 853b7e5d-76c1-490e-8269-0ef31ba2ff13
 caps.latest.revision: 10
 ---
 # AccessDBProviderSample01
+
 This sample shows how to declare a provider class that derives directly from the [System.Management.Automation.Provider.Cmdletprovider](/dotnet/api/System.Management.Automation.Provider.CmdletProvider) class. It is included here only for completeness.
 
 ## Demonstrates
 
-> [!IMPORTANT]
```

Comment:
What happened to this content?

Reply:
This was trimmed out by my code and I re-ran it against the old version and fixed it.